zachsinterview/src/main/java/com/imprev/zachsinterview/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.imprev.zachsinterview;
+
+public class Main {
+
+    /**
+     * Demonstrates how to connect to thirdPartyAPI and tests how the numerical key store may be used.
+     */
+    public static void main(String[] argv) throws NumberStoreException {
+        
+        String username = "moxiworks";
+        String password = "fjdskal3kd!jdkdInterviewFun!!!";
+        String apiUrl = "https://example.com/thirdPartyApiURL";
```

Comment:
might be good to split this into different subdomains (test.example.com, prod.example.com), separate http vs https (also https is usually port 443 but we're running on port 80), and separate path (/thirdPartyApiUrl) for ease of configuration